src/landing_game_object.py

```diff
@@ -50,3 +55,15 @@ def set_pos(self, pos_pixel: Vec2d) -> None:
 
     def add_pos_to_dict(self):
         self.__dict__.update({"pos": self.pos})
+
+    @property
+    def name(self) -> str:
```

Comment:
Aside that name is not a desired member here, having trivial getters and setters is not desirable in python. If you do not do any checks or modifications inside either getter or setter, just leave it a public member.